### PR TITLE
Create doc about MYSQL

### LINKED ISSUE #8: Create doc about MySQL

### PR DESCRIPTION
**Pull request by swacisko**
Fixes #8

### CODE REVIEW

File: laravel.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
-# Laravel
```
Comment (kvas-damian) [inline laravel.md]: Why are you removing this file?

File: my_sql.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+DATABASES
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+* I know what kind of database MySQL is and know it's pros and cons
+* I can create a database
+* I can view database structure with basic commands
+* I know MySQL data types
+* I can create, update and delete records in the database
```
Comment (kvas-damian) [inline my_sql.md]: what about `read`?

File: my_sql.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+#### Intermediate aka _I can use it_
+* I can use constraints in database models: not null, unique, primary key, foreign key, default and index
+* I can list records meeting specified conditions using various filters (e.g. where, having, group by, order by, limit)
+* I can use joins to combine rows from different tables
+* I can use aggregate functions
+
+#### Upper-Intermediate aka _upper-intermediate_
```
Comment (kvas-damian) [inline my_sql.md:23]: Let's use level names defined in the document template: https://github.com/bethinkpl/bit-docs/blob/master/!document_template.md

File: my_sql.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+
+## Sources:
+
+[Brief documentation-like description of MySQL](https://www.w3schools.com/sql/default.asp)
+
+[Extensive tutorial for full understanding](http://www.mysqltutorial.org/)
+
+
```
Comment (kvas-damian) [inline my_sql.md]: Do we need so many empty lines in this file?

File: my_sql.md (hunk 3 above)
Comment (kvas-damian) [inline my_sql.md:23]: I think "usage and understanding of the `explain` statement" should be placed at this level.

File: laravel.md (hunk 1 above)
Comment (swacisko) [reply laravel.md]: I removed it to leave only mysql.md in that pull request.

Comment (kvas-damian) [reply laravel.md]: This way, after we merge this branch the laravel doc would get removed from the repository. That's not the right path.

In this PR your task is to add docs about mysql. So there is no need to modify/delete laravel. When it's not changed you won't see it on the Pull Request diff.
Now you have to bring back `laravel.md` to the previous state before we merge this branch to master

File: my_sql.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# MySQL
+
+
+## Prerequisite knowledge:   
+DATABASES
```
Comment (rogatty) [inline my_sql.md]:
```suggestion
databases
```
https://github.com/bethinkpl/bit-docs/blame/ddc423af19c4b224cd0b6e8f3eb8aa4458f05173/!document_template.md#L13
> In order to be able to connect topics in an automated manner, items listed in "Prerequisite knowledge" and "Next steps" sections __must__ match exact filenames of linked documents.

File: my_sql.md (hunk 2 above)
Comment (bognix) [reply my_sql.md]: I would add one more point about users in MySQL and granting access. This was the biggest problem for me when I was learning.

**Review by bognix — CHANGES_REQUESTED** [review]
Bring back laravel.md

**Review by kvas-damian — APPROVED** [review]

**Comment by kvas-damian** [comment]
Please resolve conflicts before merging.

**Pull request merged**